mllib/src/main/scala/org/apache/spark/mllib/stat/distribution/MultivariateGaussian.scala

```diff
@@ -30,33 +32,68 @@ import org.apache.spark.mllib.util.MLUtils
  * @param mu The mean vector of the distribution
  * @param sigma The covariance matrix of the distribution
  */
-private[mllib] class MultivariateGaussian(
-    val mu: DBV[Double], 
-    val sigma: DBM[Double]) extends Serializable {
+@DeveloperApi
+class MultivariateGaussian private[mllib] (
+    private[mllib] val mu: DBV[Double], 
```

Comment:
Instead of having `mu`/`sigma` private and add getters, could we make them MLlib vector/matrix types and add private members of breeze types? Then we can make this constructor public and remove getters. The overhead is little because we don't copy the data arrays.